Added billing commandlets (#371)
DarqueWarrior/vsteam: .docs/Set-VSTeamBilling.md

<!-- #include "./common/header.md" -->

> # Get-VSTeamBilling

Author: ```suggestion
# Set-VSTeamBilling
```

## SYNOPSIS

<!-- #include "./synopsis/Set-VSTeamBilling.md" -->

## SYNTAX

## DESCRIPTION

> <!-- #include "./synopsis/Set-VSTeamBilling.md" --> This cmdlet covers what can be bought on the billing tab on organization level. To buy something a special token is needed only for commercial operations.

Author: ```suggestion
<!-- #include "./synopsis/Set-VSTeamBilling.md" --> This cmdlet covers what can be bought on the billing tab on organization level. To trigger commercial actions a billing token is needed.
```

> This token can be generated by using the cmdlet `Get-VSTeamBillingToken` which is a jwt token which is based on your PAT token generated. A token can be generated by any user, but the scope of the token is defined by the users permission.

Author: ```suggestion
This token can be generated by using the cmdlet `Get-VSTeamBillingToken`. That token is a JWT-token which is based on the given PAT token. A token can be generated by any user, but the scope of the token is defined by the user's permissions.
```

In this case at least owner or project collection administrator permissions are needed.

## EXAMPLES

### Example 1: Set Microsoft hosted pipeline jobs

```powershell
$myProfile = Get-VSTeamUserProfile -MyProfile
$orgs = Get-VSTeamAccounts -MemberId $myProfile.id
Set-VSTeamBilling -Type HostedPipeline -Quantity 1 -OrganizationId $orgs[0].accountId -SubscriptionId ebd42dd8-e50e-4608-aa5d-cf0c28e9aeef
```

> Sets the concurrent jobs on organization `$orgs[0].accountId` and subscription `ebd42dd8-e50e-4608-aa5d-cf0c28e9aeef` for hosted pipelines to 1. This means depending on what was configured before either the number has ben reduced of new concurrent jobs have been bought.

Author: ```suggestion
Sets the concurrent jobs on organization `$orgs[0].accountId` and subscription `ebd42dd8-e50e-4608-aa5d-cf0c28e9aeef` for hosted pipelines to 1. This means depending on what was configured before either the number has been reduced or new concurrent jobs have been bought.
```
